src/main/java/racinggame/domain/racing/RacingRoundResults.java

```diff
@@ -0,0 +1,28 @@
+package racinggame.domain.racing;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class RacingRoundResults {
+    private final List<RacingCars> results;
+
+    public RacingRoundResults() {
+        results = Collections.unmodifiableList(new ArrayList<>());
```

Comment:
자기 생성자 호출하는 `this()`를 사용하면 어떨까요?

```suggestion
        this(new ArrayList<>());
```